translations/de.json

```
@@ -1,996 +1,1003 @@
-	"To connect to this cluster, use:  --context={{.name}}": "",
+	"To connect to this cluster, use:  --context={{.name}}": "Um zu diesem Cluster zu verbinden, verwende  --context={{.name}}",
 	"To connect to this cluster, use: kubectl --context={{.name}}": "Verwenden Sie zum Herstellen einer Verbindung zu diesem Cluster: kubectl --context = {{.name}}",
 	"To connect to this cluster, use: kubectl --context={{.name}}__1": "Verwenden Sie zum Herstellen einer Verbindung zu diesem Cluster: kubectl --context = {{.name}}",
-	"To connect to this cluster, use: kubectl --context={{.profile_name}}": "",
-	"To disable beta notices, run: 'minikube config set WantBetaUpdateNotification false'": "",
-	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\\n": "",
-	"To disable update notices in general, run: 'minikube config set WantUpdateNotification false'\\n": "",
-	"To pull new external images, you may need to configure a proxy: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/": "",
-	"To see addons list for other profiles use: `minikube addons -p name list`": "",
-	"To set your Google Cloud project,  run:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\nor set the GOOGLE_CLOUD_PROJECT environment variable.": "",
-	"To start a cluster, run: \"{{.command}}\"": "",
-	"To start minikube with Hyper-V, Powershell must be in your PATH`": "",
+	"To connect to this cluster, use: kubectl --context={{.profile_name}}": "Verwenden Sie zum Herstellen einer Verbindung zu diesem Cluster: kubectl --context={{.profile_name}}",
+	"To disable beta notices, run: 'minikube config set WantBetaUpdateNotification false'": "Um Beta-Hinweise zu deaktivieren, starte: 'minikube config set WantBetaUpdateNotification false'",
+	"To disable this notice, run: 'minikube config set WantUpdateNotification false'\\n": "Um diesen Hinweis zu deaktivieren, starte: 'minikube config set WantUpdateNotification false'\\n",
+	"To disable update notices in general, run: 'minikube config set WantUpdateNotification false'\\n": "Um Hinweise generell zu deaktivieren, starte: 'minikube config set WantUpdateNotification false'\\n",
+	"To pull new external images, you may need to configure a proxy: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/": "Um neue externe Images zu ziehen, müsste eventuell ein Proxy konfiguriert werden: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/",
+	"To see addons list for other profiles use: `minikube addons -p name list`": "Um die Addon-List für andere Profile anzusehen, verwende: `minikube addons -p name list`",
+	"To set your Google Cloud project,  run:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\nor set the GOOGLE_CLOUD_PROJECT environment variable.": "Um das Google Cloud project zu setzten,  starte:\n\n\t\tgcloud config set project <project name>\n\noder setze die Umgebungsvariabel GOOGLE_CLOUD_PROJECT.",
```

```suggestion
	"To set your Google Cloud project,  run:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\nor set the GOOGLE_CLOUD_PROJECT environment variable.": "Um das Google Cloud project zu setzten,  starte:\n\n\t\tgcloud config set project \u003cproject name\u003e\n\noder setze die Umgebungsvariabel GOOGLE_CLOUD_PROJECT.",
```
For conformity we should keep `<>` as unicode.